charts/trento-server/charts/trento-web/templates/deployment.yaml

```diff
@@ -65,9 +65,10 @@ spec:
             - name: ALERT_RECIPIENT
               value: "{{ .Values.alerting.recipient }}"
             {{ end }}
-            # TODO: actual grafana integration
+            - name: GRAFANA_PUBLIC_URL
+              value: "/grafana"
             - name: GRAFANA_API_URL
```

Comment:
Seems duplicated :thinking:

Reply:
it is duplicated, one is the config for the init container the other one is the config for the actual container, we might want to remove this later but it works as for now

Reply:
ah yes, didn't expand properly :smile:

Reply:
we could think of having a sub-template for this